python/kfserving/kfserving/storage.py

```diff
@@ -232,12 +232,23 @@ def _download_from_uri(uri, out_dir=None):
         if filename == '':
             raise ValueError('No filename contained in URI: %s' % (uri))
 
-        with requests.get(uri, stream=True) as response:
+        # Get header information from host url
+        headers = {}
+        host_uri = url.hostname
+        fields = os.getenv(host_uri, "").split(",")
+
+        for field in fields:
+            if field.startswith("header."):
+                headers[field[7:]] = os.getenv(field)
```

Comment:
is 7 the length of "header" ? Could you please put it in a a constant and use the constant here? This will be a lot more readable.